core/agent_loop.py

```
@@ -277,14 +463,42 @@ def _react_think(self, task: str) -> Dict:
+            # 提取决策和交付物
+            decisions = []
+            artifacts = []
+            
+            try:
+                start = response.find("{")
+                end = response.rfind("}") + 1
+                if start != -1 and end > start:
+                    stage_data = json.loads(response[start:end])
+                    decisions = stage_data.get("decisions", [])
+                    artifacts = stage_data.get("artifacts", [])
+            except:
```

Ruff (0.15.9)

[error] Do not use bare `except`

(E722)